tests/aws/services/lambda_/test_lambda.py

```diff
@@ -929,10 +941,15 @@ def test_lambda_url_echo_invoke(self, create_lambda_function, snapshot, aws_clie
             handler="lambda_url.handler",
         )
 
-        url_config = aws_client.lambda_.create_function_url_config(
-            FunctionName=function_name,
-            AuthType="NONE",
-        )
+        if not invoke_mode:
```

Comment:
nit: consider avoiding negation (e.g., set a default `invoke_mode = invoke_mode or InvokeMode.BUFFERED` or flip the conditional)